azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -61,10 +63,13 @@
 
   private String scannerStage = "";
 
+  // TODO kunkun-tang: Before apply guice to this class, we should make
+  // ExecutorManager guiceable.
   public TriggerManager(Props props, TriggerLoader triggerLoader,
       ExecutorManager executorManager) throws TriggerManagerException {
 
-    this.triggerLoader = triggerLoader;
+    // TODO kunkun-tang: Doing hack here to allow calling new azkaban-db code. Should fix in future.
+    this.triggerLoader = ServiceProvider.SERVICE_PROVIDER.getInstance(TriggerLoader.class);
```

Comment:
We are accepting triggerLoader as a parameter and then ignoring it. This seems a little tricky? 

Also, all the dependencies in this class are Guice injectable now. I think we can just create TriggerManager directly? Then we can get rid of `SERVICE_PROVIDER` since TriggerManager should be Guice injectable directly.